### PR TITLE
updating set reward comission

### PR DESCRIPTION
**Pull request by karthikeyan-cere**
Fixed set reward commission script, now the value will be taken from .env

### CODE REVIEW

File: scripts/validator/index.ts [hunk 1]
```diff
@@ -128,23 +133,32 @@ class Validator {
    */
   public async setCommission() {
     console.log(`\nSetting reward commission`);
+    const commission = +process.env.REWARD_COMMISSION * 10 ** 7;
```
Comment (AndreiNavoichyk) [inline scripts/validator/index.ts:136]: Move all magic numbers to constants.

Comment (karthikeyan-cere) [reply scripts/validator/index.ts:136]: @AndreiNavoichyk  This is a constant value, it is not going to change.

Comment (karthikeyan-cere) [reply scripts/validator/index.ts]: Moved to env

File: scripts/validator/index.ts [hunk 2]
```diff
@@ -179,13 +190,13 @@ async function main() {
   const validator = new Validator();
   await validator.init();
   await validator.loadAccounts();
-  await validator.generateSessionKey();
+  // await validator.generateSessionKey();
```
Comment (AndreiNavoichyk) [inline scripts/validator/index.ts:193]: If you don't need the code remove it not comment.

File: scripts/validator/index.ts [hunk 3]
```diff
@@ -128,23 +133,32 @@ class Validator {
    */
   public async setCommission() {
     console.log(`\nSetting reward commission`);
+    const commission = +process.env.REWARD_COMMISSION * +process.env.REWARD_COMMISSION_MULTIPLIER;
```
Comment (AndreiNavoichyk) [inline scripts/validator/index.ts:136]: What is the Multiplier. What does it mean? Why do we need it? It should be clear from variable name.

File: scripts/validator/.env [hunk 4]
```diff
@@ -5,4 +5,5 @@ CONTROLLER_ACCOUNT_JSON=CONTROLLER_ACCOUNT_JSON
 CONTROLLER_ACCOUNT_PASSWORD=CONTROLLER_ACCOUNT_PASSWORD
 BOND_VALUE=999900000000000000
 REWARD_COMMISSION=5
+COMM_MUL=10000000
```
Comment (AndreiNavoichyk) [inline scripts/validator/.env:8]: It is not going to be changed. No sense to put it in env file. Move it back to code.

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**